EnglishFilter/sections/specific.txt

```diff
@@ -2173,7 +2173,7 @@ gunafrica.co.za##.banners-box
 gunafrica.co.za##.header-banner-cont
 kitty-kats.net##.siropuShoutboxFooter
 mathway.com###static-ad-right-alt
-yourarticlelibrary.com,cocowest.ca,torrents-proxy.com,gadgetguideonline.com##div[class^="code-block code-block-"][style^="margin: 8px"]
+otvfoco.com.br,yourarticlelibrary.com,cocowest.ca,torrents-proxy.com,gadgetguideonline.com##div[class^="code-block code-block-"][style^="margin: 8px"]
```

Comment:
I don't see any other languages except Portuguese. You can move the rule to the AdGuard Spanish.